### PR TITLE
Add types for Amplifier

### PR DESCRIPTION
**Pull request by superheri**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/amplifier/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": false,
```
Comment (armanio123) [inline types/amplifier/tsconfig.json]: Could you please enable back the strictness? Or is there any particular reason why to disabled it?

File: types/amplifier/tslint.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "ban-types": false,
```
Comment (armanio123) [inline types/amplifier/tslint.json]: Is there a reason why disabling this rules?

**Review by armanio123 — CHANGES_REQUESTED** [review]

File: types/amplifier/tsconfig.json (hunk 1 above)
Comment (superheri) [reply types/amplifier/tsconfig.json]: It is disabled in @types/amplify too so I don't see the difference.

I tried enabling it but a lot of tests are failing.

File: types/amplifier/tslint.json (hunk 2 above)
Comment (superheri) [reply types/amplifier/tslint.json]: No idea, i copied the file from @types/amplify

**Review by armanio123 — APPROVED** [review]

**Comment by armanio123** [comment]
Thanks for working on the comments. Merging.

**Pull request merged**